packages/cdk/directives/src/initial-focus/initial-focus.directive.ts

```diff
@@ -1,13 +1,16 @@
 import { AfterViewInit, Directive, ElementRef, Input, NgZone, OnDestroy } from '@angular/core';
-import { Nullable } from '@angular-ru/cdk/typings';
+import { Subject, timer } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
 
 const MIN_DELAY: number = 500;
 
 @Directive({ selector: '[initialFocus]' })
 export class InitialFocusDirective implements AfterViewInit, OnDestroy {
     private readonly className: string = 'initial-focused';
-    private timeoutId: Nullable<number> = null;
-    @Input() public focusDelay: number = MIN_DELAY;
+    private readonly unsubscribe$: Subject<boolean> = new Subject<boolean>();
+    @Input('focusDelay') public delay: number = MIN_DELAY;
+    @Input('focusDisabled') public disabled: boolean = false;
+    @Input('type') public type?: string;
```

Comment:
`@Input() public type?: string;`

Reply:
@Lightllord check it

Reply:
did it like that just to not change variables in the logic
fixed